src/mpl/src/timer/mpl_timer_gethrtime.c

```diff
@@ -75,6 +75,8 @@ int MPL_wtime_init(void)
     if (is_initialized)
         goto fn_exit;
 
+    is_initialized = 1;
+
```

Comment:
I find this a bit silly since it is just an empty function; but I guess consistency is not bad.